### PR TITLE
[admission-controller] Add option to not create rbac roles

### PR DESCRIPTION
**Pull request by anderssonw**
## What this PR does / why we need it:

Currently the admission controller does not allow you do not create rbac roles for the webhook. Our team wants to create these clusterroles ourselves to not over-privilege the SA we have for sysdig, and therefore this is a useful addition.

Could be an idea to do this globally for the admission controller instead of separately for the webhook and scanner.

## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Title of the PR starts with chart name (e.g. [mychartname])
- [x] Chart Version bumped for the respective charts
- [x] Changelog updated for the charts with version updates.
- [x] Variables are documented in the README.md (or README.tpl in some charts)
- [x] Check GithubAction checks (like lint) to avoid merge-check stoppers

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/admission-controller/values.yaml [hunk 1]
```diff
@@ -119,6 +119,10 @@ webhook:
     # Configure port for the webhook service
     port: 443
 
+  rbac:
+    # Whether to create a clusterrole and bind it
```
Comment (legobrick) [inline charts/admission-controller/values.yaml]: This and README.md description in table need to be the same. Mind changing this to `Enable the creation of ClusterRoles and the binding of these roles` ?

**Review by legobrick — CHANGES_REQUESTED** [review]

File: charts/admission-controller/values.yaml (hunk 1 above)
Comment (anderssonw) [reply charts/admission-controller/values.yaml]: Done :)

**Comment by anderssonw** [comment]
Not entirely sure why the pre-commit fails. I attempted to run it locally and the linting is fine, but not sure how to run the docs.

**Comment by dark-vex** [comment]
Thanks @anderssonw for your contribution! We added a couple of more changes to this other PR #664 we have cherry-picked your commits, so you're still the initial contributor :)

**Pull request closed without merging**